cmd/branchProtection.go

```diff
@@ -185,6 +189,17 @@ func (verifier BranchProtectionVerifier) createProtectionRequest(requireSonar bo
 	}
 }
 
+func createRequiredStatusChecks(requiredChecks []string) *github.RequiredStatusChecks {
+	if len(requiredChecks) > 0 {
+		return &github.RequiredStatusChecks{
+			Strict:   true,
+			Contexts: requiredChecks,
+		}
+	} else {
+		return nil
```

Comment:
Why do we need nil here? Can' we return the RequiredStatusChecks with empty context list?

Reply:
Sorry, that's not possible with the GH API